lib/galaxy/model/__init__.py

```diff
@@ -204,8 +204,12 @@ def system_user_pwent(self, real_system_username):
             except KeyError:
                 pass
         else:
```

Comment:
I think you should test for `real_system_username is not None`. Or is this function never called if real_system_username is not configured?